geometry_msgs/Point[] base_path
duration[] time_from_start
